### DISCUSSION TITLE
面向新兴领域的开源知识工程

### DISCUSSION THREAD

**Discussion by giscus** [zhuangbiaowei/zhuangbiaowei.github.io#5] [Announcements]
# 面向新兴领域的开源知识工程

从知识工程到知识图谱

https://zhuangbiaowei.github.io/2022/12/18/open-source-knowledge-engineering-for-emerging-areas.html

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)
 等等,忘记问了,为什么选择了 mdbook 这个 Rust SSG ?
